framework/src/executioners/FEProblemSolve.C

```diff
@@ -71,7 +71,7 @@ FEProblemSolve::validParams()
   params.addParam<unsigned int>("nl_max_funcs", 10000, "Max Nonlinear solver function evaluations");
   params.addParam<Real>("nl_abs_tol", 1.0e-50, "Nonlinear Absolute Tolerance");
   params.addParam<Real>("nl_rel_tol", 1.0e-8, "Nonlinear Relative Tolerance");
-  params.addParam<Real>("nl_div_tol", -1, "Nonlinear Divergence Tolerance");
+  params.addParam<Real>("nl_div_tol", 1.0e10, "Nonlinear Divergence Tolerance");
```

Comment:
It  might be helpful to put some words here to describe how we came up this magic number

Reply:
"We just guessed a number we thought wouldn't disrupt converging models but would save solves that were really bad"

Reply:
But then I would dismiss your review :cry:

Reply:
Then do not worry.